Description: Remove execute flag from data files
 execution permission is not required for files under
 the /usr/share/neofetch/ascii/distro directory.
Forwarded: yet
Last-Update: 2016-11-19
Author: Nobuhiro Iwamatsu <iwamatsu@debian.org>

diff --git a/Makefile b/Makefile
index 324332f..71def67 100644
--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@ install:
 	$(INSTALL_PROG) neofetch $(DESTDIR)$(PREFIX)/bin/neofetch
 	$(INSTALL_FILE) neofetch.1 $(DESTDIR)$(PREFIX)/share/man/man1/neofetch.1
 	$(INSTALL_PROG) config/config $(DESTDIR)$(PREFIX)/share/neofetch/config
-	$(INSTALL_PROG) ascii/distro/* $(DESTDIR)$(PREFIX)/share/neofetch/ascii/distro
+	$(INSTALL_FILE) ascii/distro/* $(DESTDIR)$(PREFIX)/share/neofetch/ascii/distro
 
 uninstall:
 	$(RM) $(DESTDIR)$(PREFIX)/bin/neofetch
